tools/BuildAssets/BuildTasks/Microsoft.Azure.Sdk.Build.Tasks/SdkBuildTasks/ArchiveSymbolsTask.cs

```diff
@@ -0,0 +1,522 @@
+                        }
+                        else
+                        {
+                            _assemblyList = new List<string>();
+                        }
+                    }
+                }
+
+                return _assemblyList;
+            }
+        }
+
+        /// <summary>
+        /// In Jenkins ${BUILD_NUMBER} provides the JobId
+        /// </summary>
+        public string BuildJobId
+        {
+            get
+            {
+                if(string.IsNullOrWhiteSpace(_buildJobId))
+                {
+                    Random r = new Random(DateTime.Now.Minute);
+                    _buildJobId = string.Format("rnd{0}", r.Next(99999).ToString());
+                }
+
+                return _buildJobId;
+            }
+
+            set
+            {
+                _buildJobId = value;
+            }
+        }
+
+        /// <summary>
+        /// True: Will not execute to queue up symbols request, will only create the request file and copy symbols to the SysmbolArchiveRootDir
+        /// False: Will create request file as well as execute to queue up symbol archive request
+        /// </summary>
+        public bool SkipExecuteSymbolsRequest { get; set; }
+        
+        /// <summary>
+        /// Project name registered with Symbol Archiving service
+        /// If not provided default name will be used set by the Azure .NET SDK team
+        /// </summary>
+        public string ArchiveSymbolsRequestProjectName { get; set; }
+
+        public ArchiveSymbolsTask()
+        {
+            SymbolsRequestFileList = new List<string>();
+            SkipExecuteSymbolsRequest = false;
+        }
+
+        public override bool Execute()
+        {
+            TaskLogger.LogInfo("Executing Archiving Task.....");
+
+            List<SymbolsCopyInfo> symbolList = GetSymbolsInfo();
+
+            foreach(SymbolsCopyInfo symInfo in symbolList)
+            {
+                symInfo.CreateSymbolRequest();
+                SymbolsRequestFileList.Add(symInfo.SymbolRequestFilePath);
+
+                if(SkipExecuteSymbolsRequest == false)
+                {
+                    symInfo.SendSymbolRequest();
+                    
+                    if(!string.IsNullOrWhiteSpace(symInfo.ArchiveRequestErrorOutput))
+                    {
+                        ApplicationException aEx = new ApplicationException(symInfo.ArchiveRequestErrorOutput);
+                        TaskLogger.LogException(aEx);
+                    }
+                    else
+                    {
+                        TaskLogger.LogInfo(symInfo.ArchiveRequestOutput);
+                    }
+                }
+            }
+
+            return true;
+        }
+
+        private List<SymbolsCopyInfo> GetSymbolsInfo()
+        {
+            // Symbol archive Path format: 12-12-2018\BuildJobId-4DigitRandomNumber\PackageName
+            int randomMaxValue = 9999;
+            Random rnd = new Random(DateTime.Now.Second);
```

Comment:
I'd use the default constructor for Random.

Reply:
Will make the change as suggested.